library/Director/PropertyModifier/PropertyModifierGetHostByAddr.php

```diff
@@ -29,7 +29,8 @@ public function getName()
 
     public function transform($value)
     {
-        $host = gethostbyaddr($value);
+        if (is_null($value)) { return null; }    
+	$host = gethostbyaddr($value);
```

Comment:
Please respect [PSR-2](http://www.php-fig.org/psr/psr-2/) - you can use [phpcs](https://github.com/squizlabs/PHP_CodeSniffer) to check your syntax. And please use `$value === null` instead of `is_null`.